entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EntityAdminController.java

```diff
@@ -72,6 +82,7 @@
 
         LOG.info("Permanently deleting entityId={}", entityRecord.getEntityId());
         entityRecordService.delete(entityRecord.getEntityId());
+        solrService.deleteById(entityRecord.getEntityId());
```

Comment:
we have to do exception handling here as well, but for admins we probably want to have a slightly different behaviour, eventually to indicate in response that the entity was deleted in mongo but failed to be deleted in solr